jira/client.py

```diff
@@ -3945,31 +3952,31 @@ def create_project(
 
         ps_list: List[Dict[str, Any]]
 
-        if not permissionScheme:
+        if permissionScheme is None:
```

Comment:
while more correct this slightly changes the API => changes are then not only a patch but also a change of functionality. Although I think that people which have an impact by this change were using the library wrong anyway :-)

I also notice these args are not in the docstring 
added https://github.com/pycontribs/jira/issues/1132 for future reference